wkconnect/backends/wkw/backend.py

```diff
@@ -14,15 +15,19 @@
 
 
 class Wkw(Backend):
+    repo: DatasetRepository
+
     def __init__(self, config: Dict, http_client: ClientSession) -> None:
         super().__init__(config, http_client)
+        cache_size = config.get("dataCacheSize", 1000)
```

Comment:
What’s the unit here? File handles? Bytes? Buckets? Could this be encoded in the config key?